#[
SyntaxError: Cannot anonymously call resulting type 'Integer'.
    from bad_anon_call.lly:6
]#

(10)()
